### ISSUE TITLE
[Bug]: MAPI - /downloadUrl sometimes returns an incorrect URL

### ISSUE THREAD

**Issue by Goncharo** [metriport/metriport#313]
### Steps to reproduce

TL;DR apparently sometimes an invalid URL is returned from MAPI when querying `/downloadUrl`:

`2023-05-09T21:12:35.774-04:00 ERROR --- [nio-8400-exec-4] .b.i.m.DefaultMetriportConnectionService : Raw downloadUrl response: {"url":"https://s3.us-west-2.amazonaws.com/"}`

This supposedly happens the second time this workflow is ran in rapid succession (ie ms):

- Trigger a document query
- Wait for the webhook payload to come in
- Send a request to /downloadUrl with the URL in the webhook payload (issue here with the response)

(This is a user report and no attempts to reproduce have been successful so far).

### What did you expect to happen?

A presigned URL is returned.

### What is actually happening?

Some URL pointing to nothing is returned - `"https://s3.us-west-2.amazonaws.com/"`

### Environment

```shell
Sandbox
```

### Any additional comments?

Next steps suggestion from @leite08 -> "we could modify use the `mock-webhook.ts` to be called by MAPI and try to download a file"

**Comment by Goncharo** [comment]
Have never seen this since the initial report - closing
